ql/src/java/org/apache/hadoop/hive/ql/ddl/view/CreateViewDesc.java

```diff
@@ -70,21 +72,28 @@
   private String storageHandler; // only used for materialized views
   private Map<String, String> serdeProps; // only used for materialized views
   private Set<String> tablesUsed;  // only used for materialized views
+  private List<String> sortColNames;  // only used for materialized views
+  private List<FieldSchema> sortCols;  // only used for materialized views
```

Comment:
You are right, this is not really required. We only use it to print the column names together with their types in the explain for `CREATE MATERIALIZED VIEW` statement. Logic is similar to partition columns, this will only be needed if we store these columns as entries in the SORT / BUCKET columns in metastore (as we do for normal tables), but currently we do not do that.